package.json

```diff
@@ -82,6 +82,7 @@
     "istanbul-instrumenter-loader": "^3.0.1",
     "istanbul-lib-coverage": "^3.0.0",
     "json-server": "^0.16.3",
+    "lodash.clonedeep": "^4.5.0",
```

Comment:
Will see about removing this dependency, although I think I started doing it in a PR pending review...